assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss

```diff
@@ -50,31 +50,36 @@
 		}
 
 		.googlesitekit-km-widget-tile__metric-change-container {
-			align-items: flex-start;
 			display: flex;
-			gap: 8px;
-			@media (min-width: $bp-nonMobile) {
-				align-items: center;
-			}
 		}
 	}
 
 	.googlesitekit-km-widget-tile--numeric {
+		height: 100%;
+
 		.googlesitekit-km-widget-tile__metric {
 			font-family: $f-secondary;
 			font-size: $fs-headline-lg;
+			line-height: $lh-headline-lg;
 			margin-top: -8px;
 			@media (min-width: $bp-nonMobile) {
 				margin: 0;
 			}
 		}
 
 		.googlesitekit-km-widget-tile__subtext {
-			margin: 2px 0 0;
+			margin: -2px 0 1px;
```

Comment:
Hmm, according to [Figma](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/Key-Metrics?type=design&node-id=7450-14642&mode=design&t=WbWMAmg6j4XmlhoZ-0), this subtext element should be using the `label/small` font size/line height at the mobile viewport, i.e. `font-size: $fs-label-sm; line-height: $lh-label-sm`. It should also have font weight 500 (`$fw-medium`), as should the title:

![image](https://github.com/google/site-kit-wp/assets/18395600/3589a09f-22f5-4188-b035-1977c19828df)